src/linux/cgroups2.cpp

```diff
@@ -487,6 +487,27 @@ Try<set<pid_t>> processes(const string& cgroup)
     pids.insert(*pid);
   }
 
+  if (!recursive) {
+    return pids;
+  }
+
+  Try<set<string>> cgroups = cgroups2::get(cgroup);
+  foreach (const string& cgroup, *cgroups) {
+    Try<set<pid_t>> _pids = cgroups2::processes(cgroup);
+    if (_pids.isError() && cgroups2::exists(cgroup)) {
+      return Error(
+          "Failed to get processes in '" + cgroup + "': " + _pids.error());
+    } else if (_pids.isError()) {
+      // The cgroup was just removed. Ignore the error and treat it as if
+      // the cgroup had no processes.
+      continue;
```

Comment:
this is going to ignore parse failures as well since you're looking at any error from the processes call, would be better to check this on read